### ISSUE TITLE
add cryptsetup to preinstalled packages

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-iso#14]
In order to facilitate the addition of LUKS capabilities in future Rockstor versions I propose that we add the following package and it's dependencies to our default install list:
```
cryptsetup
```
This will then ease the introduction of LUKS full disk encryption management currently in development under the following issues:
https://github.com/rockstor/rockstor-core/issues/1494
and in turn:
https://github.com/rockstor/rockstor-core/issues/550

The named package contains the facilities to format and manage LUKS devices and is available from the "base" repo.

**Comment by phillxnet** [comment]
@schakrava Given that:
https://github.com/rockstor/rockstor-core/pull/1622 "enhance disk role/management subsystem. Fixes #1494"
has now been merged I would like to request that this issue be addressed in the next iso release as then we are set to have our consequent testing channel updates ready for:

https://github.com/rockstor/rockstor-core/issues/550 "Support full disk LUKS #550"

which I intend to make my next major focus.

**Comment by phillxnet** [comment]
@schakrava given:
```
yum deplist rockstor | grep cryptsetup
  dependency: cryptsetup
   provider: cryptsetup.x86_64 1.7.2-1.el7
```
does this mean we are sorted on this issue?
I.e. Rockstor package now pulls in cryptsetup via dependency, not so on 3.8.16-16 rpm included on 3.9.0 iso but as from 3.9.0-1 rpm package:
```
yum install rockstor-3.9.0-1
...
---> Package rockstor.x86_64 0:3.8.16-16 will be updated
---> Package rockstor.x86_64 0:3.9.0-1 will be an update
...
Dependencies Resolved

===========================================================================
 Package         Arch        Version           Repository             Size
===========================================================================
Updating:
 rockstor        x86_64      3.9.0-1           Rockstor-Testing       15 M
Installing for dependencies:
 cryptsetup      x86_64      1.7.2-1.el7       base                  124 k

Transaction Summary
===========================================================================
Install             ( 1 Dependent package)
Upgrade  1 Package

Total download size: 15 M
```
and given an “Encrypt my data. You’ll set a passphrase next.” install option installs cryptsetup we are assured it is on the ISO in order to meed this install's requirements.

So it looks like going forward we can close this issue?

**Comment by schakrava** [comment]
That's right @phillxnet Thanks for verifying this.
